dynamic_array.py

```diff
@@ -0,0 +1,41 @@
+# Hacker Rank Problem Link: https://www.hackerrank.com/challenges/dynamic-array/problem
+
+#!/bin/python3
+
+import os
+import sys
+
+#
+# Complete the dynamicArray function below.
+#
+def dynamicArray(n, queries):
+    #
+    # Write your code here.
+    #
+    lastAns = 0
+    seqList = [[]]* n
+    for query in queries:
+        if query[0] == 1:
+            seqIndex  = (query[1] ^ lastAns) % n
+            seqList[seqIndex] = seqList[seqIndex] + [query[2]]
+        elif query[0] == 2:

```

Comment:
This doesn't handle invalid input (and you should *always* assume that any user input is untrusted).

Ex bogus line: 3 1 2